test/openAPI/features/support/helpers/helpers.js

```diff
@@ -1,7 +1,181 @@
+                              field_id: { type: 'integer' },
+                            },
+                            additionalProperties: false,
+                          },
+                        },
+                      },
+                      additionalProperties: false,
+                    },
+                  },
+                  primaryKey: { type: 'boolean' },
+                  readOnly: { type: 'boolean' },
+                  description: { type: 'string' },
+                  example: { type: 'string' },
+                  id: { type: 'integer' },
+                },
+                additionalProperties: false,
+              },
+              Child: {
+                type: 'object',
+                properties: {
+                  type: {
+                    type: 'string',
+                    properties: {
+                      type: 'object',
+                      properties: {
+                        ID: {
+                          type: 'object',
+                          properties: {
+                            type: { type: 'string' },
+                            description: {
+                              type: 'string',
+                            },
+                            example: {
+                              type: 'string',
+                            },
+                            id: {
+                              type: 'integer',
+                            },
+                          },
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+            additionalProperties: false,
+          },
+          incrementIndex: {
+            type: 'integer',
+          },
+          required: {
+            type: 'array',
+            items: {
+              type: 'string',
+            },
+          },
+          additionalProperties: false,
+        },
+      },
+      schema_tags: {
+        type: 'array',
+        items: {
+          type: 'object',
+          properties: {
+            name: { type: 'string' },
+            path: { type: 'string' },
+            is_fulltext: { type: 'boolean' },
+          },
+          additionalProperties: false,
+        },
+      },
+      schema_flags: {
+        type: 'array',
+        items: {
+          type: 'object',
+          properties: {
+            name: { type: 'string' },
+            path: { type: 'string' },
+          },
+        },
+      },
+      fields_uniques: {
+        type: 'array',
+        items: {
+          type: 'array',
+          items: {
+            type: 'string',
+          },
+        },
+      },
+      is_draft: { type: 'boolean' },
+      is_disabled: { type: 'boolean' },
+      is_archived: { type: 'boolean' },
+      modified_at: { type: 'string' },
+      by_user_name: { type: 'string' },
+      by_user_auth_id: { type: 'integer' },
+      by_on_behalf_of_user_auth_id: { type: 'integer' },
```

Comment:
those 2 fields have as example ```null```
![image](https://user-images.githubusercontent.com/119664031/226933735-b861b67f-d0cf-4eb5-8a00-ca2cb824576b.png)

What will happen if we want an integer, but server will send null?

Reply:
Type integer cannot be null like in an example field. @ingmarvali please let me know if spec should be update and we need to add type integer or null, or just update an example field to number, e.q. "0" or "1" etc.?

Reply:
Please add what you think is the best solution.

Reply:
Does this example help:
{
  "id": 0,
  "version": "string",
  "name": "string",
  "description": "string",
  "institution": "string",
  "number_format": "string",
  "schema": {
    "additionalProp1": "string",
    "additionalProp2": "string",
    "additionalProp3": "string"
  },
  "schema_tags": [
    {
      "name": "string",
      "path": "string",
      "is_fulltext": true
    }
  ],
  "schema_flags": [
    {
      "name": "string",
      "path": "string"
    }
  ],
  "fields_uniques": {
    "additionalProp1": "string",
    "additionalProp2": "string",
    "additionalProp3": "string"
  },
  "is_draft": true,
  "is_disabled": true,
  "is_archived": true,
  "modified_at": "2023-03-22T20:50:12.337Z",
  "by_user_name": "string",
  "by_user_auth_id": 0,
  "by_on_behalf_of_user_auth_id": 0,
  "by_on_behalf_of_user_name": "string",
  "generic_services": [
    {
      "service_id": 0,
      "name": "string",
      "is_visible": true,
      "used_count": 2147483647
    }
  ],
  "data_index_increment": 9223372036854776000,
  "has_logo": true
}

Reply:
@ingmarvali Yes, thank you! I'm going to also update the spec - change some of examples so they're not confusing ;)